career/working-and-collaborating-remotely.md

```diff
@@ -0,0 +1,172 @@
+
+> 1995: The information superhighway will mean anyone can do anything from anywhere
+>
+> 2015: must be willing to relocate to San Francisco
+
+The quote references a job posting by a tech company that produces software for remote collaboration! It appears they prefer their workers onsite! Because many companies are testing the value of remote work and there is a lack of visual interactions between coworkers, one must be cognizant of how to navigate remote work and collaboration professionally and how to optimize the use of the tools available for professionalism and professional growth.
+
+#### Working from home or a rented office
+- **Work/Life balance** - As shown in the "2019 Survey of remote workers" blog post, one of the main struggles for remote workers is disconnecting from work. When work and life begin to mix together it is very important to have:
+  - A dedicated area(s) for where you do your work
+  - A set of rituals that are done before and after work - such as getting dressed as if going into the office and walking around the block for a "commute", before sitting down at the work station
+  - A separate computer or account to separate work from personal computer time
+  - A discussion about expectations with family or roommates regarding your work time, and a plan as to when and how to interact with others that may share the same household
+- **Time management and productivity** - Another struggle closely related to work/life balance is time management. This  problem arises because as opposed to being around others in a traditional office, there are no social queues for break and lunch times and an environment where you see everyone quiet and working. Tips:
+  - Have a clean and uncluttered work area
+  - Set a specific weekly and daily schedule for work time
+  - Plan your day with a task list
+  - Use tools like Pomodoros and timers
+  - Make your own Kanban board for micromanaging smaller parts of your tickets
+  - Avoid "busy work" and focus on your assignments
+  - Turn off unimportant notifications and leave the TV for "life" hours
+  - Block time to get in the coding zone, list your other todos that aren't immediate and block time for those separately
+  - A proper diet - one can keep a journal and monitor their energy levels according to what they eat to see what's best  
+  - Plan meals - make a specific lunch time as opposed to snacking
+  - Exercise - many tech jobs are completely sedentary, exercise can boost productivity and morale
+
+* **Ergonomics** - Just like any other working situation, it is vitally important to maintain good ergonomics while working remotely.  However, the responsibility will likely be on you, the maintainer of your working space, to take the lead on either assessing your work station, or, working with a supervisor or HR to have someone from your business assess your workstation
```

Comment:
Can you link a part of this to our existing ergo lesson (using a relative link):

https://github.com/Techtonica/curriculum/blob/master/onboarding/ergonomics.md

Reply:
Done, see below